<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpgAdvanced</class>
 <widget class="QWidget" name="GpgAdvanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel7">
        <property name="text">
         <string>Generate key:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtGenKey"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel8">
        <property name="text">
         <string>List public keys:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtPublic"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel9">
        <property name="text">
         <string>List secret keys:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtSecret"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TextLabel10">
        <property name="text">
         <string>Import public:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edtImport"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="TextLabel11">
        <property name="text">
         <string>Export public:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="edtExport"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="TextLabel13">
        <property name="text">
         <string>Encrypt:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="edtEncrypt"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="TextLabel14">
        <property name="text">
         <string>Decrypt:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="edtDecrypt"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
